Tests/StoreKitUnitTests/StoreKit2/StoreKit2TransactionListenerTests.swift

```diff
@@ -217,11 +217,13 @@ class StoreKit2TransactionListenerTransactionUpdatesTests: StoreKit2TransactionL
 
     @available(iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4, *)
     func testNotifiesDelegateForRenewals() async throws {
+        self.testSession.timeRate = .oneRenewalEveryTwoSeconds
         try await self.simulateAnyPurchase(finishTransaction: true)
 
         await self.listener.listenForTransactions()
 
-        try? self.testSession.forceRenewalOfSubscription(productIdentifier: Self.productID)
+        // swiftlint:disable:next force_try
+        try! await Task.sleep(nanoseconds: 2 * 1_000_000_000)
```

Comment:
Nit: it might be good to sleep for a little longer than 2 seconds to give the renewal a little bit of breathing room to finish before verifying

```suggestion
        try! await Task.sleep(nanoseconds: 2 * 1_100_000_000)
```